pkg/urlutil/url.go

```diff
@@ -5,6 +5,8 @@ import "strings"
 var validUrlPrefixes = []string{
 	"http://",
 	"https://",
+	"tcp://",
+	"udp://",
```

Comment:
@tiborvass this will def relax IsURL so I'm gonna do that simple Prefix checks here